### PR TITLE
Add pawn.json for sampctl

### PR DESCRIPTION
**Pull request by seanny**
I added a pawn.json to be able to work with sampctl.

### CODE REVIEW

File: pawn.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "user": "maddinat0r",
+    "repo": "samp-discord-connector",
+    "resources": [
+        {
+            "name": "^discord-connector-(.*)-Linux.tar.gz$",
```
Comment (maddinat0r) [inline pawn.json]: The regex is incorrect, you need to escape the dots in the file ending.

File: pawn.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            "platform": "linux",
+            "archive": true,
+            "includes": ["pawno/include"],
```
Comment (maddinat0r) [inline pawn.json]: The paths for the include and plugin binary is incorrect. The packaged release archive contains a root folder where those files are in. You're also missing the `log-core` library.
Check the MySQL plugin pawn.json for further reference: https://github.com/pBlueG/SA-MP-MySQL/blob/master/pawn.json

Comment (seanny) [reply pawn.json]: I've updated everything, do you think this will work?

**Comment by maddinat0r** [comment]
Looks good, merged. Thanks!

**Comment by MichaelBelgium** [comment]
@seanny @maddinat0r There's a small mistake: 

`"plugins": ["discord-connector-.+\\/plugins\\/discord-connector\\.so"],`

should be 
`"plugins": ["discord-connector-.+\\/plugins\\/discord-connector.so"],`

Same with the .dll most likely. When i execute `sampctl server ensure`
it downloads `discord-connector\.so` to my plugins folder, so thats the cause.

**Pull request merged**